src/Modules/Unix/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.psd1

```diff
@@ -2,7 +2,7 @@
 GUID="EEFCB906-B326-4E99-9F54-8B4BB6EF3C6D"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="� Microsoft Corporation. All rights reserved."
+Copyright="� Microsoft Corporation. All rights reserved."

```

Comment:
Please retain the original character. This unintentional change is usually done by VSCode.